core/src/main/scala/spinal/core/FixedPoint.scala

```diff
@@ -23,6 +23,17 @@ package spinal.core
 import scala.collection.mutable.ArrayBuffer
 import spinal.core._
 
+trait  RoundType ;
+object Ceil           extends RoundType ;// RoundUp
+object Floor          extends RoundType ;// RoundDown
+object FloorToZero    extends RoundType ;// RoundToZero
+object CeilToInf      extends RoundType ;// RoundToInf
+object RoundUpp       extends RoundType ;// RoundHalfUp    ; RoundUp->RoundUpp Temporary distinguish with roundUp
```

Comment:
@Dolu1990 `RoundUp` conflict to `misc.roundUp`(Scala seems to be case insensitive),  i temporary move to `RoundUpp`, it's not good , Do you have any better suggestions?

Reply:
That's one thing i don't like about Scala, enum like stuff often conflict together.
Now i'm always putting enum like stuff into a enclosing object to avoid that hell ^^

```scala

trait RoundType
object RoundType{
  object Floor          extends RoundType ;// RoundDown
  object FloorToZero    extends RoundType ;// RoundToZero
  object CeilToInf      extends RoundType ;// RoundToInf
  object RoundUpp       extends RoundType ;
} 

```

Would be fine ?